pkg/transform/kubernetes/kubernetes.go

```diff
@@ -411,32 +411,32 @@ func CreateK8sObjects(app *spec.App) ([]runtime.Object, error) {
 	objects = append(objects, pvcs...)
 	log.Debugf("app: %s, pvc: %s\n", app.Name, spew.Sprint(pvcs))
 
-	return objects, nil
+	return objects, app.ExtraResources, nil
 }
 
-func Transform(app *spec.App) ([]runtime.Object, error) {
+func Transform(app *spec.App) ([]runtime.Object, []string, error) {
```

Comment:
Same here as for `CreateK8sObjects`  return values of those function are getting more complicated and it would be good to have a comment explaining it.